tcms/rpc/tests/test_testexecutionstatus.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=attribute-defined-outside-init, invalid-name, objects-update-used
+
+from tcms.rpc.tests.utils import APITestCase
+from tcms.tests.factories import TestExecutionFactory
+
+
+class TestFilter(APITestCase):
+    """Test TestExecutionStatus.filter"""
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        # populate database with TestExecutions
+        self.execution_1 = TestExecutionFactory()
+        self.execution_2 = TestExecutionFactory()
+
+        self.execution_status_1 = self.execution_1.status
+        self.execution_status_2 = self.execution_2.status
+
+    def test_filter_statuses(self):
+        execution_statuses = self.rpc_client.TestExecutionStatus.filter(
+            {"pk__in": (self.execution_status_1.pk, self.execution_status_2.pk)}
```

Comment:
Probably most distinguished feature of execution statuses is their weight: negative, neutral or positive:
https://kiwitcms.readthedocs.io/en/latest/admin.html#test-execution-statuses

To validate the API works correctly it is enough to filter by a certain weight and assert that the result is not an empty list. (or not even filter at all).

There are 6 statuses inserted via migrations already, see `testruns/migrations/0004_squashed.py` so no setup is needed for the test.